testing/test_pluginmanager.py

```diff
@@ -3,7 +3,12 @@
 """
 import pytest
 import types
-import importlib_metadata
+
+try:
```

Comment:
Would you mind changing this to:

```python
if sys.version_info >= (3, 8):
    from importlib import metadata
else:
    import importlib_metadata as metadata
```

?

This makes it explicit which version this is supposed to work (so we can eventually find and remove the obsolete code in the future when <py37 support is dropped), and also avoids hiding unrelated bugs in broken environments which could cause `from importlib import metadata` to raise an `ImportError` even on Python 3.8.

Reply:
I could, however I always thought that feature detection is preferred.

Reply:
I prefer the explicit version because of the aforementioned reasons. It was a breeze to remove Python 2.7 only features when they were explicitly marked using `six.PY2` or `sys.version_info`, but the ones using feature detection were much harder (and I doubt we got them all, probably there are others in place still).

Reply:
What if I add a comment there? (Just to the actual source, not to tests and docs).

As for broken environments, that would be broken anyway, as importlib_metadata would not be installed.

Reply:
however, I can definitively change that, yes.

Reply:
> What if I add a comment there? (Just to the actual source, not to tests and docs).

If you don't mind I would prefer the explicit version check, sorry for being insistent on that. 😁 

> As for broken environments, that would be broken anyway, as importlib_metadata would not be installed.

Yes, but then we will get a message like "importlib not available because <broken environment>" (the real problem), instead of "Could not import importlib_metadata", which would be hide the original error and would be baffling (experienced this myself more than once unfortunately).

Reply:
amended